src/deluge/deluge.cpp

```diff
@@ -451,17 +451,19 @@ void setupBlankSong() {
 
 /// Can only happen after settings, which includes default settings, have been read
 void setupStartupSong() {
-	auto templatePath = "SONGS/DEFAULT.XML";
 	auto startupSongMode = FlashStorage::defaultStartupSongMode;
 
-	if (startupSongMode == StartupSongMode::BLANK) {
-		setupBlankSong();
-	}
-	else if (startupSongMode == StartupSongMode::TEMPLATE && !storageManager.fileExists(templatePath)) {
-		setupBlankSong();
-		currentSong->writeTemplateSong(templatePath);
-	}
-	else {
+	switch (startupSongMode) {
+	case StartupSongMode::TEMPLATE:
+	case StartupSongMode::LASTOPENED:
```

Comment:
```suggestion
	case StartupSongMode::TEMPLATE:
	    [[fallthrough]];
	case StartupSongMode::LASTOPENED:
		[[fallthrough]];
```

Reply:
what's that for ?

Reply:
Tells the compiler that the fall through is intentional, not the result of forgetting a break;